rpc/handlers.go

```diff
@@ -58,6 +58,9 @@ var (
 
 	// These errors can be only be returned by Juno-specific methods.
 	ErrSubscriptionNotFound = &jsonrpc.Error{Code: 100, Message: "Subscription not found"}
+
+	// TODO[pnowosie]: Update the error while specification describe it
+	ErrBlockNotRecentForProof = &jsonrpc.Error{Code: 1001, Message: "Block is not sufficiently recent for storage proofs"}
```

Comment:
What do you think about responding that the user should try set Block ID to "latest"?